### PR TITLE
New package: opensubdiv

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

File: var/spack/repos/builtin/packages/opensubdiv/package.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    implement high performance subdivision surface (subdiv)
+    evaluation on massively parallel CPU and GPU architectures.
+    This code path is optimized for drawing deforming surfaces
+    with static topology at interactive framerates."""
+
+    homepage = "http://graphics.pixar.com/opensubdiv/docs/intro.html"
+    url      = "https://github.com/PixarAnimationStudios/OpenSubdiv/archive/v3_4_0.tar.gz"
+    git      = "https://github.com/PixarAnimationStudios/OpenSubdiv"
+
+    version('dev', branch='dev')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/opensubdiv/package.py]: You should call this version develop if you want it to sort properly

File: var/spack/repos/builtin/packages/opensubdiv/package.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    version('3_4_0',     sha256='d932b292f83371c7518960b2135c7a5b931efb43cdd8720e0b27268a698973e4')
+
+    variant('tbb', default=False, description='Builds with Intel TBB support')
+    variant('openmp', default=False, description='Builds with OpenMP support')
+    variant('doc', default=False, description='Builds documentation. Requires python@@2.6')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/opensubdiv/package.py]: Why the double @?

Comment (qwertos) [reply var/spack/repos/builtin/packages/opensubdiv/package.py]: `@` needs to be escaped with `@@` else `==> Error: Incomplete color format: '@ in python@2.6` for command `spack info opensubdiv`

File: var/spack/repos/builtin/packages/opensubdiv/package.py (hunk 1 above)
Comment (qwertos) [reply var/spack/repos/builtin/packages/opensubdiv/package.py]: fixed

File: var/spack/repos/builtin/packages/opensubdiv/package.py (hunk 2 above)
Comment (adamjstewart) [reply var/spack/repos/builtin/packages/opensubdiv/package.py]: Hmm, that's probably a bug. Want to open up a separate issue for this? In the meantime, you can just change it to "Python 2", or not mention the dependency at all.

Comment (qwertos) [reply var/spack/repos/builtin/packages/opensubdiv/package.py]: fixed

**Pull request merged**